test/sql/catalog/sequence/test_duckdb_sequences.test

```diff
@@ -0,0 +1,17 @@
+# name: test/sql/catalog/sequence/test_duckdb_sequences.test
+# group: [sequence]
+
+require noforcestorage
```

Comment:
This has no-force-storage because we save the `usage_count` but not the `last_value`
In `duckdb_sequences` this now thinks that `last_value` is set

Reply:
On top of that, since we store the last_value as start value, when the sequence is stored, the value will be one higher